mllib/src/test/scala/org/apache/spark/ml/classification/OneVsRestSuite.scala

```diff
@@ -101,6 +101,45 @@ class OneVsRestSuite extends SparkFunSuite with MLlibTestSparkContext with Defau
     assert(expectedMetrics.confusionMatrix ~== ovaMetrics.confusionMatrix absTol 400)
   }
 
+  test("one-vs-rest: tuning parallelism does not change output") {
```

Comment:
Is there a good way to do that?  I'm having trouble thinking of ways to do it which would not produce flaky tests.